src/NHibernate/Action/IAsyncExecutable.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NHibernate.Action
+{
+	public interface IAsyncExecutable : IExecutable
```

Comment:
I would add a `// 6.0 TODO: merge into IExecutable`. Or is there any value in keeping a separated interface for async methods?